x-pack/plugins/fleet/public/components/uninstall_command_flyout/empty_policy_name_hint.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiIcon, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const EMPTY_POLICY_NAME_HINT = i18n.translate(
+  'xpack.fleet.uninstallTokenList.emptyPolicyNameHint',
+  {
+    defaultMessage: 'The related agent policy has already been deleted, hence its name is unknown.',
+  }
+);
+
+export const EmptyPolicyNameHint = () => (
```

Comment:
this looks good, Ash! thanks for the suggestion, in case there will be no objection soon, I'll include it

<img width="622" alt="image" src="https://github.com/elastic/kibana/assets/39014407/08e0d319-4c86-4219-adc4-9d11589b4d7b">

<img width="472" alt="image" src="https://github.com/elastic/kibana/assets/39014407/a6ad36df-d36f-4fbd-b220-e86e34d8cc5f">

Reply:
Thanks @gergoabraham . Yeah the row with just an icon looks very different from the other rows and a user might miss it thinking it is just a grouping/divider icon. I'm hoping with a `-`. it is slightly more visible. I think we could also show a `No name available` in italics or a subdued colour maybe be even better.

Reply:
added here: 703c519e73bbebcd2c622c555214fd797786f83c

Reply:
I think once the user reads the hint next to the dash, it will be clear to them what the dash means (probably even before that), so it's better not pollute the screen with 'No name available' strings, there could be a lot of them which can be distractive. I'd go with the dash - what do you think?

Reply:
Yeah either is fine with me.